src/config/layers.js

```diff
@@ -379,8 +380,8 @@ export const gemeindegrenzen = new TrafimageGeoServerWMSLayer({
       },
       tileGrid: new TileGrid({
         extent: projectionExtent,
-        resolutions,
-        matrixIds: resolutions.map((r, i) => `${i}`),
+        resolutions: LayerHelper.mapResolutions,
+        matrixIds: LayerHelper.mapResolutions.map((r, i) => `${i}`),
```

Comment:
Move map resolutions to LayerHelper

Reply:
Why not use the new helper function for all layers referencing `resolutions` and delete the `resolutions` variable in this file?

Reply:
I have replaced `mapResolutions` by `getMapResolutions()`